docs/faq.md

```diff
@@ -143,19 +143,31 @@ RHCOS/OCP version | RHEL version
 
 ## Q: How do I determine what version of an RPM is included in an RHCOS release?
 
-Since OpenShift 4.12, the operating system is shipped as a bootable container image.  See [layering docs](https://docs.openshift.com/container-platform/4.12/post_installation_configuration/coreos-layering.html#coreos-layering).
+In the future, the package list for a given OCP release will be displayed in [the OpenShift release controller](https://amd64.ocp.releases.ci.openshift.org/). Currently, only diffs are displayed.
 
-For example, you can do:
+Starting from 4.19+, it's possible to query the RPM list using `oc`:
+
+```
```

Comment:
Since this does end up pulling something that requires perms might be useful to spell add this step:

```
export REGISTRY_AUTH_FILE=/path/to/pull-secret
```